src/vlogger/vlogger.cpp

```diff
@@ -75,9 +75,13 @@ namespace log_level
 	static const char *log_names_warn[]    = {"warn",  "warning", "2", NULL};
 	static const char *log_names_info[]    = {"info",  "information", "3", NULL};
 	static const char *log_names_details[] = {"details", NULL};
+#if !defined(VMA_OPTIMIZE_LOG) || (VMA_OPTIMIZE_LOG > 5)
 	static const char *log_names_debug[]   = {"debug", "4", NULL};
+#endif
+#if !defined(VMA_OPTIMIZE_LOG) || (VMA_OPTIMIZE_LOG > 6)
```

Comment:
It is not the default case as we allow the user to set 2 parameters with contradictions (while each parameter individually is valid). In this case - need to issue a warning.